DigitalLearningSolutions.Web/Helpers/ExternalApis/MapsApiHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers.ExternalApis
+{
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.Models.External.Maps;
+    using Newtonsoft.Json;
+
+    public interface IMapsApiHelper
+    {
+        Task<MapsResponse> GetPostcodeCoordinates(string postcode);

```

Comment:
It looks like instead of getting coordinates, we're getting MapsResponse. It might be worth making the current method private and then add a public method that returns a lat/long tuple?